### ISSUE TITLE
[6.x][NPC] Darnassus Sentinel not giving directions

### ISSUE THREAD

**Issue by GnaXi** [TrinityCore/TrinityCore#17414]
**Description**:
Darnassus Sentinel not giving directions

**Current behaviour**:
When speaking to a Darnassus Sentinel in Darnassus, no directions are given.

**Expected behaviour**:
The Darnassus Sentinels should be giving directions to almost everything in Darnassus

**Steps to reproduce the problem**:
1. Go to Darnassus
2. Speak to a Darnassus Sentinel

**Branch(es)**:
6.x

**TC hash/commit**:
5fdeb0667532

**TDB version**:
6.04

**Operating system**:
Debian 8

![darnassussentinel](https://cloud.githubusercontent.com/assets/1616448/16170193/a56ea41c-354a-11e6-9828-effbb5d5fd29.png)

**Comment by Anddrw** [comment]
Same goes for every guard in stormwind, or other capital city.

**Comment by LightbringerX** [comment]
Correct me if I'm wrong but isn't this a global phenomenon in 6.x as this occurs all around the world (of warcraft) atm?

**Comment by Aokromes** [comment]
Stormwind gossips fixed on https://github.com/TrinityCore/TrinityCore/commit/6239339069e2390ee3a08cbdc5c5c1d18148407b
Orgrimmar guard gossip https://github.com/TrinityCore/TrinityCore/commit/59ee37691e263bc4465c11e76a086929188cdb70
Thunder Bluff guard gossips fixed on
https://github.com/TrinityCore/TrinityCore/commit/05e0643b0515d245956e3182d2312762d4778389

**Comment by Aokromes** [comment]
Darnassus and Teldrasil fixed on 7b39a05

**Comment by Aokromes** [comment]
Dun Morogh and Goldshire fixed on 1beb0a4

**Comment by Aokromes** [comment]
Ironforge fixed on 850390c

**Comment by funjoker** [comment]
assign that to me.
Not done with my big updates yet. Still ongoing

**Comment by matanshukry** [comment]
@funjoker what's the status on this / what is currently missing ?

**Comment by meji46** [comment]
Stormwind Guard (Goldshire): https://gist.github.com/meji46/272ad6f971c62654b973a45637a1b521

**Comment by Northernn** [comment]
# guards (orgrimmar) menu fix (directions)
UPDATE creature_template SET gossip_menu_id='1951' WHERE entry='3296';

**Comment by meji46** [comment]
> # guards (orgrimmar) menu fix (directions)
> UPDATE creature_template SET gossip_menu_id='1951' WHERE entry='3296';

As I said in this issue https://github.com/TrinityCore/TrinityCore/issues/28014
> The only problem is that gossip_menu_id is 0 in the table creature_template. I'm updating all POI and gossip menus of the guards (and I have this bug fixed). I'll open an issue with the sql when I finish it.

I'm working on all the guards adding all the missing information related to the pois

**Comment by meji46** [comment]
> https://gist.github.com/shamage/66fbf8b52f14d20f3a6ba86227038cda

I'm already working on the POIs for all the zones and I have all the Orgrimmar stuff corrected. Your fix is also incorrect because conditions are missing and the whole gossip_menu part needs to be done manually (you're adding broadcast_text ids, not npc_text ids)

**Comment by shamage** [comment]
yes, I already noticed it ... the old POIs need to be deleted, there are a lot of them
thank you for the instruction
